src/resources/js/tribe-events.js

```diff
@@ -865,11 +865,11 @@ Date.prototype.format = function( mask, utc ) {
 				$tribe_bar.tribe_clear_form();
 			}
 
-			params = tribe_ev.fn.parse_string( params );
+			params = tribe_ev.fn.parse_string( decodeURIComponent( params.replace( /\+/g, '%20' ) ) );
```

Comment:
Looking at the contents of `value`, it appears that that variable is an array. So doing something like the following would be handy:

```js
if ( Array.isArray( value ) ) {
  value = decodeURIComponent( value[0].replace( /\+/g, '%20' ) );
}
```